test/NomoPlayersDropMechanic.test.ts

```diff
@@ -757,7 +811,10 @@ describe("NomoPlayersDropMechanic tests", function () {
         deployerAddress,
         tokenPrice,
         maxQuantity,
-        collectibleItems) as NomoPlayersDropMechanic;
+        vrfCoordinatorAddress,
```

Comment:
Since we are using the `collectibleItems` as length, consider removing the deployment and use the one in the `beforEach`